pkg/controllers/provisioning/scheduling/suite_test.go

```diff
@@ -493,6 +493,18 @@ var _ = Describe("Topology", func() {
 			Expect(env.Client.List(ctx, &nodes)).To(Succeed())
 			ExpectSkew(ctx, env.Client, "default", &topology[0]).To(ConsistOf(2, 2, 1))
 		})
+		It("should match all pods when labelSelector is nil", func() {
+			topology := []v1.TopologySpreadConstraint{{
+				TopologyKey:       v1.LabelTopologyZone,
+				WhenUnsatisfiable: v1.DoNotSchedule,
+				LabelSelector:     nil,
```

Comment:
IIUC, you can just omit this line for the same effect.

Reply:
I was going for explicitly what I wanted the value to be :)